tools/WinObjC.Tools/vsimporter-templates/frameworks.txt

```diff
@@ -0,0 +1,61 @@
+# WinObjC.Frameworks
+Accelerate.lib
+Accounts.lib
+AddressBook.lib
+AddressBookUI.lib
+AdSupport.lib
+AssetsLibrary.lib
+AudioToolbox.lib
+AudioUnit.lib
+AutoLayout.lib
+AVFoundation.lib
+AVKit.lib
+CloudKit.lib
+Contacts.lib
+CoreAudio.lib
+CoreAudioKit.lib
+CoreBluetooth.lib
+CoreData.lib
+CoreGraphics.lib
+CoreImage.lib
+CoreLocation.lib
+CoreMedia.lib
+CoreMIDI.lib
+CoreMotion.lib
+CoreTelephony.lib
+CoreText.lib
+CoreVideo.lib
+EventKit.lib
+EventKitUI.lib
+GameController.lib
+GameKit.lib
+GamePlayKit.lib
+GLKit.lib
+HealthKit.lib
+HomeKit.lib
+iAd.lib
+ImageIO.lib
+LocalAuthentication.lib
+MapKit.lib
+MediaAccessibility.lib
+MediaPlayer.lib
+MessageUI.lib
+Metal.lib
+OpenGLES.lib
+QuartzCore.lib
+QuickLook.lib
+SafariServices.lib
+Security.lib
+Social.lib
+StoreKit.lib
+SystemConfiguration.lib
+Twitter.lib
+UIKit.lib
+UIKit.Xaml.lib
```

Comment:
Yes.

---
In reply to: [110000107](https://github.com/Microsoft/WinObjC/pull/2418#discussion_r110000107) [](ancestors = 110000107)